### DISCUSSION TITLE
[Kavita] Age Rating order

### DISCUSSION THREAD

**Discussion by crass00** [Kareadita/Kavita#2634] [Ideas]
### Idea Description

Hi,

I think the age rating order in Kavita needs a small change.  Currently If I select "M" rating, that will show "MA+" comics as well.  Going by the interface in Kavita I can see that the order as the drop down shows it so it makes sense in that way within Kavita.  However and this might just be becuase I'm from Australia M is  recommended for 15 year olds MA is restricted to 15+.  I'd have left it there cuase every country is going to have there own idea on this, but checking comic tagger they have M before MA+ in age  rating ordered list for age rating so maybe i'm not the only one :)

### Duration of Using Kavita

1 year

### Idea Category

User Experience

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
So this is something unlikely to be changed. The age rating order comes from the order that ComicRack came up with and at the moment, all reading softwares are binded to it. I agree, the list of age ratings is very verbose, confusing, and the ordering, like MA15 being after M, makes no sense. 

But making a new set of ratings or reordering would break parity and could be confusing for the end user.
